testing/trino-testing-services/src/main/java/io/trino/testng/services/Listeners.java

```diff
@@ -37,4 +40,24 @@ public static void reportListenerFailure(Class<? extends ITestNGListener> listen
         // Therefore, instead of throwing, we terminate the JVM.
         System.exit(1);
     }
+
+    public static String formatTestName(ITestClass testClass)
```

Comment:
Could you please extract changes to logging into a separate commit?

Reply:
done